docs/changelog/5_x.rst

```diff
@@ -0,0 +1,343 @@
+====
+v5.0
+====
+
+:edb-alt-title: EdgeDB v5
+
+To play with the new features, install the CLI using `our installation guide
+<https://www.edgedb.com/install>`_ and initialize a new project.
```

Comment:
For the duration of prerelease, we must tell them to install nightly.

I think it is plausible that it will get merged by friday. See https://github.com/edgedb/edgedb-cli/pull/1217